src/wl-clipboard.cpp

```diff
@@ -0,0 +1,48 @@
+#include "wl-clipboard.h"
+
+#include <QProcess>
+#include <QStandardPaths>
+#include <QTextCodec>
+
+#include "logger.hpp"
+
+const QString wl_copy_path = QStandardPaths::findExecutable("wl-copy");
+const QString wl_paste_path = QStandardPaths::findExecutable("wl-paste");
```

Comment:
More elegant would be to initialize it when it needed. 

Suggestion:

```c++
bool wl_copy_to_clipboard(const QString& text) {
    static const QString wl_copy_path = QStandardPaths::findExecutable("wl-copy");
    ...
}

bool wl_paste_clipboard(QString& outText) {
    static const QString wl_paste_path = QStandardPaths::findExecutable("wl-paste");
    ...
}
```

`static` guarantees that initialization occurs only once and not every time the function is called.